### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by Eternal-Flame085**
- Created facade
- Created facade spec
- Refactored all URL's to their paths

### CODE REVIEW

File: app/services/movie_api_service.rb [hunk 1]
```diff
@@ -22,8 +22,9 @@ def self.movie_search(query)
 
     2.times do
       page_number += 1
-      response = conn.get("/3/search/movie?&page=#{page_number}") do |req|
+      response = conn.get("/3/search/movie") do |req|
```
Comment (stillsheryl) [inline app/services/movie_api_service.rb:25]: Great improvement on the readability.

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -14,6 +14,7 @@
 #
 require 'simplecov'
 SimpleCov.start 'rails'
+SimpleCov.add_filter ['app/jobs', 'app/channels', 'app/helpers']
```
Comment (stillsheryl) [inline spec/spec_helper.rb:17]: Good thing to remember for the future!

**Review by stillsheryl — APPROVED** [review]
Nice job refactoring with the facade and cleaning up the path names!

File: app/views/viewing_parties/new.html.erb [hunk 3]
```diff
@@ -6,9 +6,9 @@
   <p>
     Movie Title  <%= @movie.title %>
   </p>
-  <%= form_with url: '/viewing-party/create', local: true do |f| %>
+  <%= form_with url: viewing_party_create_path, local: true do |f| %>
     <%= f.label :duration_of_party %>
-    <%= f.text_field :duration_of_party, value: @movie.runtime %><br>
+    <%= f.number_field :duration_of_party, value: @movie.runtime, min: @movie.runtime %><br>
```
Comment (Stearnzy) [inline app/views/viewing_parties/new.html.erb:11]: Thanks for changing this!

File: spec/facade/search_facade_spec.rb [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+  it "movie_search can return array of hashes with 40 search results" do
+    VCR.use_cassette('movie_search') do
+      @movies = SearchFacade.movie_search("star")
+      expect(@movies.class).to eq(Array)
+      expect(@movies[0].class).to eq(Hash)
+
+      expect(@movies[0][:title]).to eq("Star Wars")
+      expect(@movies[39][:title]).to eq("The Sun Is Also a Star")
+    end
```
Comment (Stearnzy) [inline spec/facade/search_facade_spec.rb:23]: Reminder that we should change this to objects

**Review by Stearnzy — APPROVED** [review]

**Pull request merged**